### PR TITLE
Ignore servers in serverlist that specify an invalid player count

### PR DESCRIPTION
**Pull request by Laupetin**
Filter out any server that answer with a server info that specifies a player count that is greater than 18 or lower than 0.

### CODE REVIEW

**Review by diamante0018 — APPROVED** [review]
Tested it, it works.

**Review by Rackover — APPROVED** [review]
Very nitpicky: don't we have a macro for max player count?
Shouldn't we have one?

otherwise it's good, and can be merged as soon as you want to :)

File: src/Components/Modules/ServerList.cpp [hunk 1]
```diff
@@ -472,6 +472,11 @@ namespace Components
 				// Remove server from queue
 				i = ServerList::RefreshContainer.servers.erase(i);
 
+				// Servers with more than 18 players or less than 0 players are faking for sure
+				// So lets ignore those
+				if(server.clients > 18 || server.maxClients > 18 || server.clients < 0 || server.maxClients < 0)
```
Comment (mjkzy) [inline src/Components/Modules/ServerList.cpp:477]: add space between ``if`` and the ``(``?

**Review by mjkzy — CHANGES_REQUESTED** [review]

File: src/Components/Modules/ServerList.cpp (hunk 1 above)
Comment (Laupetin) [reply src/Components/Modules/ServerList.cpp:477]: and i thought i was nitpicky lmao

**Comment by Laupetin** [comment]
> Very nitpicky: don't we have a macro for max player count?
> Shouldn't we have one?
> 
> otherwise it's good, and can be merged as soon as you want to :)

Not macro. The game for sure has one the client doesn't though.

**Review by Dss0 — APPROVED** [review]

**Pull request merged**